Tests/Realm.Tests/Database/AsyncTests.cs

```diff
@@ -46,6 +46,7 @@ public void AsyncWrite_ShouldExecuteOnWorkerThread()
 
                 Assert.That(_realm.All<Person>().Count(), Is.EqualTo(1));
                 Assert.That(otherThreadId, Is.Not.EqualTo(currentThreadId));
+                Assert.That(false);
```

Comment:
Is this PR still a draft and you use this to test the new GH action or is there some deeper meaning behind that line that yet have to understand? :)

Reply:
It's still a draft but I opened it by mistake as "ready for review" and until you push again you can't set it as draft (at least so I think). Then after pushing, I forgot to set it to "draft". I am just testing how error reporting looks.

Reply:
Ahhh, gotcha! Just wanted to make sure I understand everything that's going on. :)